git_machete/cmd.py

```diff
@@ -391,11 +379,11 @@ def get_root_of(b: str) -> str:
         print(bold("Discovered tree of branch dependencies:\n"))
         self.status(warn_on_yellow_edges=False)
         print("")
-        do_backup = os.path.isfile(self.definition_file_path)
-        backup_msg = f"\nThe existing definition file will be backed up as {self.definition_file_path}~" if do_backup else ""
-        msg = f"Save the above tree to {self.definition_file_path}?{backup_msg}" + pretty_choices('y', 'e[dit]', 'N')
-        opt_yes_msg = f"Saving the above tree to {self.definition_file_path}... {backup_msg}"
-        ans = ask_if(self.cli_opts, msg, opt_yes_msg)
+        do_backup = os.path.isfile(self._definition_file_path)
```

Comment:
Hmm AFAICS `_definition_file_path` and `_managed_branches` are preceded with a single underscore... is it deliberate?

Reply:
Ok, `@property`